ingestion-core/src/main/java/com/mozilla/telemetry/ingestion/core/transform/AddMetadata.java

```diff
@@ -31,7 +31,8 @@ private AddMetadata() {
 
   private static final String HEADER = "header";
   private static final List<String> HEADER_ATTRIBUTES = ImmutableList //
-      .of(Attribute.DATE, Attribute.DNT, Attribute.X_DEBUG_ID, Attribute.X_PINGSENDER_VERSION);
+      .of(Attribute.DATE, Attribute.DNT, Attribute.X_DEBUG_ID, Attribute.X_PINGSENDER_VERSION,
+          Attribute.X_SOURCE_TAGS);
```

Comment:
@relud: Should we be removing most of the logic from `ingestion-beam`'s `AddMetadata` in favor of the `core` implementation?

Reply:
yeah, i'll file a PR on that

Reply:
https://github.com/mozilla/gcp-ingestion/pull/1330

Reply:
You should be leaving my account alone.